packages/url/README.md

```diff
@@ -167,4 +167,19 @@ const newUrl = removeQueryArgs( 'https://wordpress.org?foo=bar&bar=baz&baz=fooba
 Removes one or more query string arguments from the given URL.
 
 
+### safeDecodeURI
+
+```js
+const badUri = safeDecodeURI( '%z' ); // does not throw an Error, simply returns '%z'
+```
```

Comment:
Can you put a line break after this code block? Also I think in `does not throw an Error` should error be lowercase?

Reply:
Added the whitespace, but I'd like to leave Error capitalized since it's referencing a JavaScript [Error object](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error)

Reply:
@georgeh Oh yes of course, that makes sense! Should it be capitalised in the description below too then?